crates/gitbutler-branch-actions/src/dependencies.rs

```diff
@@ -24,7 +28,7 @@ pub fn compute_workspace_dependencies(
     let mut stacks_input: Vec<InputStack> = vec![];
     for stack in stacks {
         let mut commits_input: Vec<InputCommit> = vec![];
-        // Commit IDs in application order
+        // The commit id's must be ordered from order they get applied.
```

Comment:
It feels a bit like there's something funny in this sentence, would it be clearer with this? `// Commit id's must be ordered tail first.`

Reply:
Hmm, tail introduces more questions imo.
How about talking about parentage?
`// Commit id's must be ordered ancestors-to-descendants`
or 
`// Commit id's are expected to be ordered parents first.`?